lib/mix/tasks/check.ex

```diff
@@ -165,13 +165,15 @@ defmodule Mix.Tasks.Check do
   Task will load the configuration in following order:
 
   1. Default stock configuration.
-  2. `.check.exs` in user home directory.
-  3. `.check.exs` in current project directory (or umbrella root for an umbrella project).
+  2. `.check.exs` in `--config-directory` opt on command line.
```

Comment:
Personally I feel that it's quite unnatural to point to config directory instead of directly to a config file. I believe that most tools that have similar option allow to specify a file and not a dir. It also works more naturally with shell auto-completion. Aaand that'd be more in line with what you've originally suggested in #16, isn't it? What do you think?

Btw, coding that should not be that much more work - you'd have to change `load_from_dirs` to `load_from_files` and join `@config_filename` to directories in `load` instead of in `load_from_dirs`.

Reply:
Yeah I went with the directory since that was the tenor of the other code, not wanting to rock the boat too much. I agree that it pointing to the file seems more natural. Will change. :+1: